config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh

```diff
@@ -2,14 +2,14 @@
 
 set -xeuo pipefail
 
-brctl show
-
-base_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+#base_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+base_dir="$( pwd )"
 
 function clean_up {
     set +e
     ara generate junit - > /home/output/logs/ansible_xunit.xml
     if [ -e "$base_dir/hosts" ]; then
+        virsh screenshot --file /home/output/logs/atomic-host-fedoraah.ppm atomic-host-fedoraah
```

Comment:
any reason for this name: fedoraah ?

Reply:
not really.  Originally I was doing this as a block: rescue: piece in ansible and that would be the name if I was using {{ item.key }}.  I can make it just atomic-host.ppm, would that be better?

Reply:
not a big deal...atomic-host.ppm is fine.